spec/numberformat.html

```diff
@@ -618,6 +618,23 @@
       </emu-alg>
     </emu-clause>
 
+    <emu-clause id="sec-intl.numberformat.prototype.formattoparts">
+      <h1>Intl.NumberFormat.prototype.formatToParts ( [ _value_ ] )</h1>
```

Comment:
in the `DateTimeFormat.prototype.formatToParts` we decided to make the argument non-optional. https://tc39.github.io/ecma402/#sec-Intl.DateTimeFormat.prototype.formatToParts, this should do the same.